docs/administrators/operational_procedures.rst

```diff
@@ -18,31 +18,42 @@ This process describes the steps to prepare an inmanta project for production re
 Context
 ++++++++
 * The project development and testing is complete
-* All modules have been properly released.
+* All modules have been properly released (See :ref:`Releasing and distributing modules` for the procedure).
 * The project is in a git repo, with a specific branch dedicated to production releases
 * The project is checked out on disk.
 * All modules are checked out on the correct, tagged commit.
 
 Procedure
 ++++++++++
 
-1. Verify in `project.yml` that `install_mode` is set to `release`.
-2. Freeze all modules with
+1. Check modules install mode:
+    - For v1 modules (if any): ensure that ``install_mode`` in ``project.yml`` is set to ``release``.
+    - For v2 modules (if any): ensure that ``pip.pre`` is not set in ``project.yml``.
```

Comment:
I didn't consider this before, but the `pip` section is iso7-only. I'd leave this out.